pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter.go

```diff
@@ -82,19 +85,16 @@ func (admitter *PodEvictionAdmitter) Admit(ar *admissionv1.AdmissionReview) *adm
 	return validating_webhooks.NewPassingAdmissionResponse()
 }
 
-func (admitter *PodEvictionAdmitter) markVMI(ar *admissionv1.AdmissionReview, vmiName, nodeName string, dryRun bool) (err error) {
+func (admitter *PodEvictionAdmitter) markVMI(ar *admissionv1.AdmissionReview, vmiName, nodeName string) (err error) {
```

Comment:
IMHO here we should propagate the dryRun to the `PatchOptions` instead of getting rid of it.
As now, we are not performing the "full" dry run execution.

Reply:
> IMHO here we should propagate the dryRun to the `PatchOptions` instead of getting rid of it. As now, we are not performing the "full" dry run execution.

What will be the added value of doing so?

Reply:
The DryRun execution should perform and test all the things that will be performed by a normal execution, without storing anything, so without persisting changes.
It is used to test something.
Right now it is possible that the dryRun execution succeed, so a user thinks that everything will be good, but then the request will fail without dryRun execution due to the Patch request in the `markVMI` function.

Reply:
I will revert the dryRun to the way it was.
A follow-up PR could perform this change which is unrelated to this PR.

Reply:
Done.